internal/pkg/application/functions/functions.go

```diff
@@ -72,11 +72,9 @@ func (f *fnct) Handle(ctx context.Context, e *events.MessageAccepted, msgctx mes
 			now = now.Add(timeSinceHandleWasCalled)
 		}
 
-		// TODO: This should be persisted to a database instead
-		if loggedValues, ok := f.history[prop]; ok {
-			f.history[prop] = append(loggedValues, LogValue{Value: value, Timestamp: now})
-		} else {
-			logger.Debug().Msgf("new value was not saved to history")
+		err := f.storage.Add(context.Background(), f.ID(), prop, value, now)
```

Comment:
Varför inte skicka med ctx som anropar skickar in?

Reply:
måste missat att det fanns ett ctx redan. Ändrar.